static/js/containers/App.js

```diff
@@ -57,6 +87,48 @@ class App extends React.Component {
     }
   }
 
+  requireCompleteProfile() {
+    const {
+      userProfile,
+      userProfile: { profile },
+      location: { pathname },
+      dispatch,
+    } = this.props;
+    let profileCompleteCheck = profileComplete(profile);
+    if (
+      userProfile.getStatus === FETCH_SUCCESS &&
+      profile.agreed_to_terms_of_service &&
+      !PROFLE_REGEX.test(pathname) &&
+      profileCompleteCheck != true
+    ) {
+      const { url, title, text } = profileCompleteCheck;
+      dispatch(updateDialogText(text));
+      dispatch(updateDialogTitle(title));
+      dispatch(setDialogVisibility(true));
+      this.context.router.push(url);
+    }
+  }
+
+  showDialog () {
+    const { ui, dispatch } = this.props;
+    let visible = _.get(ui, ['dialog', 'visible']);
+    let text = _.get(ui, ['dialog', 'text']);
+    if ( visible == true && text != "" ) {
```

Comment:
Can we use triple equals here? I thought we checked this in the linter but I'm not sure